src/wizard/javascript/nextjs.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Next.js
+doc_link: https://docs.sentry.io/platforms/javascript/guides/nextjs/
+support_level: production
+type: framework
+---
+
+To use Sentry with your Next.js application, you need to use `@sentry/nextjs` (Sentry’s Next.js SDK):
+
+```bash
+yarn add @sentry/nextjs
+# or
+npm install --save @sentry/nextjs
+```
+
+To [configure your app automatically](https://docs.sentry.io/platforms/javascript/guides/nextjs/#configure) (although you can also [configure it manually](https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/)), you can run the sentry-wizard:
+
+```bash
+npx @sentry/wizard -i nextjs
+```
+
+It will automatically patch your application:
+
+- create `sentry.client.config.js` and `sentry.server.config.js` with the default `Sentry.init`.
+- create `next.config.js` with the default configuration.
+- create `sentry.properties` with configuration for sentry-cli (which is used when automatically uploading source maps).
```

Comment:
```suggestion
[Configure your app automatically](https://docs.sentry.io/platforms/javascript/guides/nextjs/#configure) with the sentry wizard. It will automatically patch your application:

- Creates `sentry.client.config.js` and `sentry.server.config.js` with the default `Sentry.init`.
- Creates `next.config.js` with the default configuration.
- Creates `sentry.properties` with configuration for sentry-cli (which is used when automatically uploading source maps).

You can also [configure it manually](https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/)).

```bash
npx @sentry/wizard -i nextjs
```
```